README.md

```diff
@@ -1 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
+# Math Magicians 🧮
+
+ Math magicians is an app for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote. 🧐
+![image](https://user-images.githubusercontent.com/31547587/158484525-6e710524-4d2e-443e-b8d8-bc95daaba965.png)
+
+## Built With 🔍
+
+- React
+
+## Authors: 👋
+
+👤 **Hector Torres**
+
+- GitHub: [@HectorTorres](https://github.com/HectorTorresE)
+- Twitter: [@HectorT](https://twitter.com/HectorT00406915)
+
+## Contributing: 🤝
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support: 🌟
+
+Give a ⭐️ if you like this project!
+
+## Getting Started: ✨
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites: 🤓
+
+- npm (node package manager)
+- Have git correctly installed.
+
+### Setup: ⚙️
+
+- Open a terminal window where you want to have installed a copy of the repository.
+
+- Use this command to get a copy:
+```
+https://github.com/HectorTorresE/math-magicians.git
+```
+
+- then, run `npm install` in your comand prompt, this will install all the listed dependencies on the package.json file.
+
+### Usage: 🎉
```

Comment:
- [ ] Let's update the readme file to include all the necessary commands, e.g, how to run the project